bin/utils.mjs

```diff
@@ -43,7 +44,7 @@ export const errorWrap =
     try {
       return await fn(...args);
     } catch (err) {
-      console.error(err);
+      Logger.getInstance().error(err);
```

Comment:
IMO Logger default export should be its default instance. This means you can have a named export `export { Logger }` and a default `export default Logger.init()`

Reply:
This would decrease the verboseness and confusion of the imports/calls.